### PR TITLE
WildfireGames.0AD version 0.0.25b

### PR DESCRIPTION
**Pull request by DanielAtKrypton**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28868)

### CODE REVIEW

File: manifests/w/WildfireGames/0AD/0.0.25b/WildfireGames.0AD.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: WildfireGames.0AD
+PackageVersion: 0.0.25b
```
Comment (ItzLevvie) [inline manifests/w/WildfireGames/0AD/0.0.25b/WildfireGames.0AD.locale.en-US.yaml:5]:
```suggestion
PackageVersion: r25860-alpha
```

File: manifests/w/WildfireGames/0AD/0.0.25b/WildfireGames.0AD.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WildfireGames.0AD
+PackageVersion: 0.0.25b
```
Comment (ItzLevvie) [inline manifests/w/WildfireGames/0AD/0.0.25b/WildfireGames.0AD.installer.yaml:5]:
```suggestion
PackageVersion: r25860-alpha
```

File: manifests/w/WildfireGames/0AD/0.0.25b/WildfireGames.0AD.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: WildfireGames.0AD
+PackageVersion: 0.0.25b
```
Comment (ItzLevvie) [inline manifests/w/WildfireGames/0AD/0.0.25b/WildfireGames.0AD.yaml:5]:
```suggestion
PackageVersion: r25860-alpha
```

File: manifests/w/WildfireGames/0AD/0.0.25b/WildfireGames.0AD.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+PackageUrl: https://play0ad.com/
+License: GPLv2
+ShortDescription: 0 A.D. is a real-time strategy video game under development by Wildfire Games.
+Description: 0 A.D. is a free, open-source, historical Real Time Strategy (RTS) game currently under development by Wildfire Games, a global group of volunteer game developers. As the leader of an ancient civilization, you must gather the resources you need to raise a military force and dominate your enemies.
+Moniker: 0AD
+Tags:
+- game
+- gaming
+- videogame
+- age of empires
```
Comment (ItzLevvie) [inline manifests/w/WildfireGames/0AD/0.0.25b/WildfireGames.0AD.locale.en-US.yaml:18]:
```suggestion
- age-of-empires
```

File: manifests/w/WildfireGames/0AD/0.0.25b/WildfireGames.0AD.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+PackageName: 0 A.D.
+PackageUrl: https://play0ad.com/
+License: GPLv2
+ShortDescription: 0 A.D. is a real-time strategy video game under development by Wildfire Games.
+Description: 0 A.D. is a free, open-source, historical Real Time Strategy (RTS) game currently under development by Wildfire Games, a global group of volunteer game developers. As the leader of an ancient civilization, you must gather the resources you need to raise a military force and dominate your enemies.
+Moniker: 0AD
+Tags:
+- game
+- gaming
+- videogame
```
Comment (ItzLevvie) [inline manifests/w/WildfireGames/0AD/0.0.25b/WildfireGames.0AD.locale.en-US.yaml:17]:
```suggestion
- video-game
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
You will need to update the version folder as well: from `0.0.25b` to `r25860-alpha`

**Review by ItzLevvie — APPROVED** [review]

File: manifests/w/WildfireGames/0AD/r25860-alpha/WildfireGames.0AD.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WildfireGames.0AD
+PackageVersion: r25860-alpha
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://releases.wildfiregames.com/0ad-0.0.25b-alpha-win32.exe
```
Comment (RokeJulianLockhart) [inline manifests/w/WildfireGames/0AD/r25860-alpha/WildfireGames.0AD.installer.yaml:9]: I've created a ticket at https://trac.wildfiregames.com/ticket/6388#comment:description to replace that with an MSIX package.

**Pull request merged**